libp2p/Capability.h

```diff
@@ -58,6 +58,9 @@ class Capability: public std::enable_shared_from_this<Capability>
 	void sealAndSend(RLPStream& _s);
 	void addRating(int _r);
 
+public:
+	uint16_t const c_protocolID;
```

Comment:
Future: Can this be moved to protected and use friend where access is necessary?